frontend/js/components/table/tableCell/generic/TableCellName.vue

```diff
@@ -1,30 +0,0 @@
-<template>
-  <a :href="editUrl"
-     v-if="!row.hasOwnProperty('deleted') && (row.edit)"
-     class="tablecell__name"
-     @click="preventEditInPlace($event)">
```

Comment:
Is the `editInModal` index option still functional after this change?

Reply:
It was not but the last commit takes care of this.